### ISSUE TITLE
Ignore custom apps

### ISSUE THREAD

**Issue by pablopunk** [pablopunk/SwiftShift#45]
Provide a UI in the settings to include other apps to be ignored

* List all file system apps?
* List open apps?

**Comment by danini-the-panini** [comment]
Can we just open a file dialog and they select the `.app` from there?

**Comment by pablopunk** [comment]
@danini-the-panini yep, that was the idea of the first bullet point. The second one is also interesting, a few apps do that. It makes sense cause you probably have the app running when you realize you want to ignore it

**Comment by txj-xyz** [comment]
Would be really nice when working in IDEs, I use `Command + Left Click` to see source in IntelliJ.

👍🏽

**Comment by nhat** [comment]
Would it be possible to prioritize this FR? I noticed there was a similar request recently, so it might be helpful to others as well.

**Comment by pablopunk** [comment]
@danini-the-panini @nhat @txj-xyz 0.27.0 is out with this feature 🥳 

<img src="https://github.com/user-attachments/assets/5b235d81-b114-43b2-882f-bd178c884f05" width="300px" />
